README.md

```diff
@@ -425,6 +425,8 @@ details them.
 
 ### `couchrestore`
 
+* `13`: restore target database is not new and empty.
```

Comment:
what about:
```suggestion
* `13`: restore target database must be a new and empty database.
```

Reply:
This doesn't match the tone of the rest error descriptions though, they are descriptive and not prescriptive, e.g. "10: backup source or restore target database does not exist." not - "10: backup source or restore target database must exist", if you see what I mean.

Reply:
Yea good point, let's keep what you have then.